docs/redirects.js

```diff
@@ -187,7 +187,6 @@ const misc = [
 /*::
 type Redirects = $ReadOnlyArray<{|
     source: string,
-    destination: string,
```

Comment:
This looks like a test that should be reverted?

Reply:
DONE